helix-rest/src/main/java/org/apache/helix/rest/server/resources/AbstractResource.java

```diff
@@ -160,6 +164,16 @@ protected Response JSONRepresentation(Object entity) {
     }
   }
 
+  protected Response JSONRepresentation(Object entity, String headerName, Object headerValue) {
```

Comment:
I suggest it should reuse the existing method `protected Response JSONRepresentation(Object entity)` to combine them, reducing duplicate code.

Reply:
TFTR. Updated.